src/schemaGenerator.ts

```diff
@@ -469,6 +478,30 @@ function addResolveFunctionsToSchema(
   checkForResolveTypeResolver(schema, requireResolversForResolveType);
 }
 
+function extendResolversFromInterfaces(schema: GraphQLSchema, resolvers: IResolvers) {
+  const typeNames = new Set([
+    ...Object.keys(schema.getTypeMap()),
+    ...Object.keys(resolvers)
+  ]);
```

Comment:
A little simpler?
```js
const typeNames = Object.keys({
  ...schema.getTypeMap(),
  ...resolvers,
});
```
Having an array rather than a `Set` should be fine because the only usage below is calling `forEach`.